src/Tizen.NUI/src/public/BaseHandle.cs

```diff
@@ -33,12 +34,18 @@ public class BaseHandle : Element, global::System.IDisposable
         /// swigCMemOwn
         /// </summary>
         /// <since_tizen> 3 </since_tizen>
+        /// When deleting the field, change it to private.
+        [Obsolete("Deprecated in API9, Will be removed in API11. Please use SwigCMemOwn instead!")]
```

Comment:
Hello, 
I can see this is for Deprecated-ACR. 
please follow the steps below.
1) add comments like : `//ToDo : will raise deprecated-ACR, [Obsolete("Deprecated in API9, Will be removed in API11. Please use SwigCMemOwn instead!")]`
2) get review in DevelNUI with No "ACR Required" tag
3) when it is merged in master branch by integration work, you can modify the comment of the [Obsolete] tag to be activated and use it for ACR process
4) the PR for ACR should be reviewed on master branch only

thank you.